tracer/src/Datadog.Trace/Headers/Ip/RequestIpExtractor.cs

```diff
@@ -0,0 +1,71 @@
+// <copyright file="RequestIpExtractor.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.Headers.Ip
+{
+    internal static class RequestIpExtractor
```

Comment:
Can you point me where it is used :prettyplease:? I found only one occurence in the next file but maybe it's in some files that I've closed.

Reply:
Indeed it's used in the `AspNetCoreHttpRequestHandler `and by the `TracingHttpModule `and by `AspNetWebApi2Integration`